.github/workflows-source/schedule-hourly.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         with:
           BAZEL_COMMAND: >-
```

Comment:
can you give context in the description and add a comment here about why this it (temporarily) needed?

Reply:
I added a comment.

Reply:
thanks, I've edited the description as well